controllers/perconaservermysql_controller.go

```diff
@@ -841,26 +955,50 @@ func (r *PerconaServerMySQLReconciler) reconcileGroupReplication(ctx context.Con
 		}
 	}
 
-	l.V(1).Info("Member state", "pod", firstPod.Name, "state", state)
+	l.V(1).Info(fmt.Sprintf("First pod state: %s", state))
 	if state == innodbcluster.MemberStateOffline {
+		l.Info("Configuring first instace", "instace", firstPodFQDN)
 		if err := mysh.ConfigureInstance(ctx, firstPodUri); err != nil {
 			return err
 		}
-		l.Info("Configured instance", "instance", firstPod.Name)
+		l.Info("Configured first instance", "instance", firstPodFQDN)
 	}
 
 	if !clusterExists {
-		if err := mysh.CreateCluster(ctx, cr.InnoDBClusterName(), firstPod.Status.PodIP); err != nil {
-			return err
+		l.Info("Creating InnoDB cluster")
+		err := mysh.CreateCluster(ctx, cr.InnoDBClusterName(), firstPod.Status.PodIP)
+		if err != nil {
+			if strings.Contains(err.Error(), "has a populated Metadata schema") {
```

Comment:
I'm not so sure about this check, maybe it is good, but I couldn't find a better way atm @hors wdyt?

Reply:
need to rework it